judge/utils/tests/test_cachedict.py

```diff
@@ -0,0 +1,20 @@
+from django.test import SimpleTestCase
+
+from judge.utils.cachedict import CacheDict
+
+
+class CacheDictTestCase(SimpleTestCase):
+    def setUp(self):
+        self.test_dict = {'a': True}
+        self.cache_dict = CacheDict(lambda id: self.test_dict[id])
+
+    def test_original_item_in_cachedict(self):
+        self.assertNotIn('a', self.cache_dict)
+        self.assertTrue(self.cache_dict['a'])
+        self.assertIn('a', self.cache_dict)
+
+    def test_new_item_in_cachedict(self):
```

Comment:
What's the point of this test case exactly? `self.cache_dict` is refreshed every case.